### ISSUE TITLE
Import classes or projects in xml format

### ISSUE THREAD

**Issue by SergeyMi37** [intersystems-community/vscode-objectscript#79]
Import classes or projects in xml format in current namespace

**Comment by isc-rsingh** [comment]
Create documentation to describe use of system management portal to do these tasks.

**Comment by gjsjohnmurray** [comment]
Possible use for customized links from server actions menu:
```
    "objectscript.conn": {
        "links": {
            "Portal Explorer": "http://${host}:${port}/csp/sys/exp/%25CSP.UI.Portal.ClassList.zen?$NAMESPACE=${ns}"
        },
        "active": true,
        "server": "local",
        "ns": "USER"
    }
```

**Comment by isc-bsaviano** [comment]
John's link has been included in the docs since #758
